FAQ.md

```diff
@@ -194,6 +242,44 @@ Check the time difference between the machine time of the monitor and the time w
 
 It is not recommended to deploy TiDB offline using Ansible. If the Control Machine has no access to external network, you can deploy TiDB offline using Ansible. For details, see [Offline Deployment Using Ansible](op-guide/offline-ansible-deployment.md).
 
+#### Fast build the cluster using Docker Compose (standalone deployment)
+
+You can use Docker Compose to build a cluster locally, including the cluster monitor. You can also customize the version and the number of instances of the components on demand and customize the configuration file which is only allowed in the development environment. For details, see [Building the Cluster Using Docker Compose](op-guide/docker-compose.md).
+
+#### How to separately record the slow query log in TiDB? How to locate the slow query SQL statement?
+
+1. The slow query definition for TiDB is in the `conf/tidb.yml` configuration file of `tidb-ansible`. `slow-threshold: 300` is the threshold value of the slow query (unit: millisecond).
+
+    The slow query log is recorded in `tidb.log` by default. If you want to generate a slow query log file separately, set `enable_slow_query_log` of the `inventory.ini` configuration file to `True`.
+
+    Then run `ansible-playbook rolling_update.yml --tags=tidb` to perform a rolling update on the `tidb-server` instance. After the update is finished, the `tidb-server` instance will record the slow query log in `tidb_slow_query.log`.
+
+2. If a slow query occurs, you can locate the `tidb-server` instance where the slow query is and slow query time point using Grafana and find the SQL statement information recorded in the log on the corresponding node.
+
+#### How to add the `label` configuration if `label` of TiKV was not configured when I deployed the TiDB cluster for the first time?
+
+The configuration of TiDB `label` is related to the cluster deployment architecture. It is important and is the basis for the global management and schedule of PD. If you did not configure `label` when deploying the cluster previously, you should adjust the deployment structure by manually adding the `location-labels` information using the PD management tool `pd-ctl`, for example, `config set location-labels "zone, rack, host"` you should configure based on the practical `label` level name. 
+
+For the usage of `pd-ctl`, see [PD Control Instruction](tools/pd-control.md).
+
+#### Why does the `dd` command for the disk test use the `oflag=direct` option?di
+
+The Direct mode wraps the Write request into the I/O command and sends this command to the disk, thus to bypass the file system cache to test the real I/O Read/Write performance of the disk.
```

Comment:
- delete "thus" (seems seldom used in tech docs)
- to test -> and directly test